step-function-demo-app/inject_inline_code.awk

```diff
@@ -0,0 +1,17 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2022 Datadog, Inc.
```

Comment:
```suggestion
# Copyright 2023 Datadog, Inc.
```

Reply:
I'm having trouble understanding this code. Maybe a comment might help

Reply:
You are not alone. This is copied from existing David's code and I didn't spend time to read the awk doc and to know the syntax 😅.

@thedavl If you already know the syntax, please feel free to chime in 😄 and I can put it in the comments. Otherwise, I'd probably read the doc when I have time later when I have more time. Thank you!